modules/accumulo/src/main/java/org/apache/fluo/accumulo/data/MutableBytes.java

```diff
@@ -91,6 +92,11 @@ public MutableBytes(String s, Charset cs) {
   }
 
   @Override
+  public final int hashCode() {
+    return Arrays.hashCode(data);
```

Comment:
In your branch, assuming you only have one commit, you could use the following commands to revert the files.  

```bash
git checkout HEAD~1 modules/api/src/main/java/org/apache/fluo/api/data/Bytes.java
git checkout HEAD~1 modules/accumulo/src/main/java/org/apache/fluo/accumulo/data/MutableBytes.java
```